### PR TITLE
Update orders-api-overview.md

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-2340](https://jira.bigcommerce.com/browse/DEVDOCS-2340)

## What changed?
Fix POST request under Adding products
Removed client id

### CODE REVIEW

File: docs/api-docs/orders/orders-api-overview.md [hunk 1]
```diff
@@ -100,7 +98,6 @@ To [get a list of order statuses](https://developer.bigcommerce.com/api-referenc
 ```http
 GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2//order_statuses
```
Comment (ektravel) [inline docs/api-docs/orders/orders-api-overview.md:99]:
```suggestion
GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2/order_statuses
```

File: docs/api-docs/orders/orders-api-overview.md [hunk 2]
```diff
@@ -144,7 +141,6 @@ Specify the [customer](https://developer.bigcommerce.com/api-reference/store-man
 ```http
 POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2//v2/orders
```
Comment (ektravel) [inline docs/api-docs/orders/orders-api-overview.md:142]:
```suggestion
POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2/orders
```

File: docs/api-docs/orders/orders-api-overview.md [hunk 3]
```diff
@@ -187,7 +182,6 @@ Add [shipping addresses](https://developer.bigcommerce.com/api-reference/store-m
...
```
Comment (ektravel) [inline docs/api-docs/orders/orders-api-overview.md:183]:
```suggestion
POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2/orders
```

File: docs/api-docs/orders/orders-api-overview.md [hunk 4]
```diff
@@ -269,12 +262,11 @@ Accept: application/json
 
 [![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/store-management/orders/orders/createanorder#requestrunner)
 
-To get the `product_options.id` and `product_options.value` of a product for the order `products` array, send a `GET` request to `/stores/{{STORE_HASH}}/v3//catalog/products/{product_id}/variants`.
+To get the `product_options.id` and `product_options.value` of a product for the order `products` array, send a `GET` request to `/stores/{{STORE_HASH}}/v3/catalog/products/{product_id}/variants`.
 
 ```http
 GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3//catalog/products/{product_id}/variants
```
Comment (ektravel) [inline docs/api-docs/orders/orders-api-overview.md:268]:
```suggestion
GET https://api.bigcommerce.com/stores/{{STORE_HASH}}/v3/catalog/products/{product_id}/variants
```

File: docs/api-docs/orders/orders-api-overview.md [hunk 5]
```diff
@@ -36,7 +36,6 @@ To [create an order](https://developer.bigcommerce.com/api-reference/store-manag
 ```http
 POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2//v2/orders
```
Comment (ektravel) [inline docs/api-docs/orders/orders-api-overview.md:37]:
```suggestion
POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/v2/orders
```

**Review by ektravel — COMMENTED** [review]
Added my suggestions

**Comment by traci707** [comment]
Thank you for finding the other issues.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**